bumble/device.py

```diff
@@ -1493,37 +1493,55 @@ async def sustain(self, timeout: Optional[float] = None) -> None:
         self.remove_listener('disconnection', abort.set_result)
         self.remove_listener('disconnection_failure', abort.set_exception)
 
-    async def set_data_length(self, tx_octets, tx_time) -> None:
+    async def set_data_length(self, tx_octets: int, tx_time: int) -> None:
         return await self.device.set_data_length(self, tx_octets, tx_time)
 
     async def update_parameters(
         self,
-        connection_interval_min,
-        connection_interval_max,
-        max_latency,
-        supervision_timeout,
-        use_l2cap=False,
-    ):
+        connection_interval_min: int,
+        connection_interval_max: int,
+        max_latency: int,
+        supervision_timeout: int,
+        use_l2cap: bool = False,
+        wait_for_complete: bool = False,
+    ) -> None:
         return await self.device.update_connection_parameters(
             self,
             connection_interval_min,
             connection_interval_max,
             max_latency,
             supervision_timeout,
             use_l2cap=use_l2cap,
+            wait_for_complete=wait_for_complete,
         )
 
-    async def set_phy(self, tx_phys=None, rx_phys=None, phy_options=None):
-        return await self.device.set_connection_phy(self, tx_phys, rx_phys, phy_options)
+    async def set_phy(
+        self,
+        tx_phys: Union[Iterable[int], int, None] = None,
+        rx_phys: Union[Iterable[int], int, None] = None,
+        phy_options: int = 0,
+        wait_for_complete: bool = False,
+    ) -> None:
+        """Sets PHY preference of this connection.
```

Comment:
Since this docstring is essentially the same as the one for `device.set_connection_phy`, maybe there's a clever way to just say "See XYZ" and refer to that other location, rather than have a copy of the docstring, which is hard to maintain in both places.